client/js/components/week/week.js

```diff
@@ -9,9 +11,16 @@ var Week = React.createClass({
   getInitialState: function () {
     return weekData;
```

Comment:
I think we should start commenting that the weekData, monthData, dayData are temporary data stubs until we get our server up and running. I will do so in my next PR, would be great if all followed suit!